ext/test/http/curl_http_test.cc

```diff
@@ -248,16 +248,46 @@ TEST_F(BasicCurlHttpTests, CurlHttpOperations)
   std::multimap<std::string, std::string, curl::curl_ci> m1 = {
       {"name1", "value1_1"}, {"name1", "value1_2"}, {"name2", "value3"}, {"name3", "value3"}};
   curl::Headers headers = m1;
-  curl::HttpOperation http_operations1(http_client::Method::Head, "/get", handler, headers, body,
-                                       true);
+  curl::HttpOperation http_operations1(http_client::Method::Head, "/get", handler,
```

Comment:
Perhaps renaming this test as `CurlHttpAsyncOperations`.

Reply:
All  previous tests are async by default. Including `http::client::SendRequest()` method. Have suffixed `Sync` explicitly where needed, keeping defaults as async.